app/project/[id]/task/[taskid]/components/task-comments.tsx

```diff
@@ -291,16 +307,17 @@ function UploadAttachment({
 
     setIsUploading(true);
 
-    assert(targetFiles.length == 1);
-    const file = targetFiles.item(0);
-    assert(file);
+    assert(targetFiles.length >= 1);
 
-    // server action arguments can only be primatives or FormData
-    const data = new FormData();
-    data.set("file", file);
-    const newFile = await addTaskFile(taskId, data);
-    // as DesignFile might be a sign of bad typing ... TODO improve
-    if (newFile) setCurrentAttachments((l) => [...l, newFile as DesignFile]);
+    const selectedFiles = Array.from(targetFiles);
+
+    for (let i = 0; i < selectedFiles.length; i++) {
```

Comment:
I think this for loop can be replaced by a `for (const file of targetFiles)` type loop

Reply:
Sure thing - sorted!